docs/using.rst

```diff
@@ -601,6 +601,12 @@ Monitor the D-Bus interface
 
   busctl monitor de.pengutronix.rauc
 
+Obtain bundle information
+
+.. code-block:: sh
+
+  busctl call de.pengutronix.rauc / de.pengutronix.rauc.Installer InspectBundle sa{sv} "/path/to/bundle" 0
```

Comment:
Perhaps this should mention that URLs are supported if streaming mode is enabled?

Reply:
Replaced by `<bundle-path>/<bundle-url>` now